packages/ui/src/components/IconButton/IconButton.tsx

```diff
@@ -48,12 +47,15 @@ export const IconButton = createComponent<IconButtonProps, 'button'>({
         {...(variantProps as RadixIconButtonProps)}
         disabled={isDisabled}
         size={size}
+        className={
+          !isDisabled && !props['aria-readonly'] ? 'cursor-pointer' : ''
+        }
       >
         {isLoading ? (
           <Spinner color="current" size={getIconSize(size, iconSize)} />
         ) : (
           <Icon
-            className={cx(iconClassName, 'cursor-pointer')}
+            className={iconClassName}
```

Comment:
you need to keep the `cx(` here right?

```suggestion
            className={cx(iconClassName)}
```

Reply:
no I don't think we need cx here.  If you look [here](https://github.com/FuelLabs/fuel-explorer/commit/dd96475bb42c8c302582d641d70ac78e658ac2bc#diff-142661a2119190194a9fe74c7977c95c8bf76d7db987d8051984701b2d53fe06) iconClassName was not initially wrapped with cx.  I believe cx is used when we want to add classes to an existing class.